infra/cifuzz/config_utils.py

```diff
@@ -125,6 +125,7 @@ def __init__(self):
     self.git_store_branch = os.environ.get('GIT_STORE_BRANCH')
     self.git_store_branch_coverage = os.environ.get('GIT_STORE_BRANCH_COVERAGE',
                                                     self.git_store_branch)
+    self.builds_storage = os.getenv('BUILDS_STORAGE')
```

Comment:
I don't really like this. I think probably this should be a bool and just use the filestore.
We should probably switch to forcing users to specify filestores, but I don't think they should specify a seperate filestore for each action.
We can have a GitAndArtifact filestore instead or something.